node/src/BaseClient.ts

```diff
@@ -3391,48 +3452,62 @@ export class BaseClient {
         key: string,
         end: Boundary<string>,
         start: Boundary<string>,
-        count?: number,
-    ): Promise<Record<string, [string, string][]> | null> {
-        return this.createWritePromise(createXRevRange(key, end, start, count));
+        options?: { count?: number } & DecoderOption,
+    ): Promise<{ entryID: GlideString; data: [string, string][] }[] | null> {
```

Comment:
Does the data not need to be a GlideString?

Reply:
Yes, it is a user data

Reply:
Update: entry ID could be given by the user, but it is still in number-like format (to be precise: `"<uint>-<uint>"`). I updated it by introducing
```ts
export type StreamEntryDataType = Record<string, [GlideString, GlideString][]>;
```
Data is user data, which could contain anything